internal/config/config.go

```diff
@@ -138,13 +153,13 @@ func (c Config) Save() error {
 	}
 
 	// Marshall into YAML
-	content, err := yaml.Marshal(c)
+	content, err := c.Marshal()
 	if err != nil {
-		return saveError{fmt.Errorf("error marshalling project configuration: %v", err)}
+		return saveError{err}
```

Comment:
why the text msg was removed?

Reply:
See [this change](https://github.com/kubernetes-sigs/kubebuilder/pull/1469/files/3596e2105d2634d2a0af7e59705918864e19ca4a#diff-0512b0d3e62f6364ee95d279bbc97aebR139).